features/main/crud.feature

```diff
@@ -523,46 +523,6 @@ Feature: Create-Retrieve-Update-Delete
     }
     """
 
-  Scenario: Update a resource with empty body
```

Comment:
It's a BC break

Reply:
Yes, but it's a bug fix. The previous behavior was wrong. We can't fix this bug without... fixing it.